isofit/radiative_transfer/radiative_transfer_engine.py

```diff
@@ -175,35 +175,34 @@ def __init__(
                     "Unknown RT mode provided in LUT file. Please use either 'transm' or 'rdn'."
                 )
 
-            # if necessary, resample prebuilt LUT to desired instrument spectral response
-            if not len(wl) == len(self.lut.wl) or all(wl == self.lut.wl):
-                Logger.info(f"Resampling LUT to instrument spectral response.")
-                conv = xr.Dataset(
-                    coords={"point": self.lut.point, "wl": wl},
-                    attrs={"RT_mode": self.rt_mode},
+            # If necessary, resample prebuilt LUT to desired instrument spectral response
+            if not len(wl) == len(self.lut.wl) or not all(wl == self.lut.wl):
```

Comment:
A put a `not` before `all(wl == self.lut.wl)`, I assume this is how it was intended to be?

Reply:
Ah yes, of course. Thanks for the catch!